docs/apache-airflow-providers-teradata/operators/teradata.rst

```diff
@@ -0,0 +1,114 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/operator:TeradataOperator:
+
+TeradataOperator
+================
+
+The purpose of TeradataOperator is to define tasks involving interactions with the Teradata.
+
+To execute arbitrary SQL in an Teradata, use the
+:class:`~airflow.providers.teradata.operators.teradata.TeradataOperator`.
+
+Common Database Operations with TeradataOperator
+------------------------------------------------
+
+Creating a Teradata database table
+----------------------------------
+
+An example usage of the TeradataOperator is as follows:
+
+.. exampleinclude:: /../../tests/system/providers/teradata/example_teradata.py
+    :language: python
```

Comment:
```suggestion
    :language: python
    :dedent: 4
```
Since the snippets that are referenced are indented in the DAG code, the snippet for the documentation should be dedented. This comment applies to the rest of the `exampleinclude` uses in this doc and the transfer doc too.